create_framework/http-kernel-httpkernelinterface.rst

```diff
@@ -8,12 +8,18 @@ goal by making our framework implement ``HttpKernelInterface``::
 
     namespace Symfony\Component\HttpKernel;
 
+    // ...
+
     interface HttpKernelInterface
     {
         /**
          * @return Response A Response instance
          */
-        function handle(Request $request, $type = self::MASTER_REQUEST, $catch = true);
+        public function handle(
```

Comment:
I'm aware of the line-length rule, but this kind of changes look awful to me. I'd prefer to keep the method declarations in one line, unless they have lots of long arguments.